hack/olm-registry/hypershift-template.yaml

```diff
@@ -0,0 +1,214 @@
+                      name: openshift-operator-lifecycle-manager
+          policyTypes:
+            - Ingress
+        - apiVersion: v1
+          kind: Service
+          metadata:
+            name: deployment-validation-operator-metrics
+            namespace: openshift-deployment-validation-operator
+            labels:
+              name: deployment-validation-operator
+          spec:
+            ports:
+            - name: http-metrics
+              port: 8383
+              protocol: TCP
+              targetPort: 8383
+            selector:
+              name: deployment-validation-operator
+        - apiVersion: v1
+          kind: ConfigMap
+          metadata:
+            name: deployment-validation-operator-config
+            namespace: openshift-deployment-validation-operator
+            labels:
+              name: deployment-validation-operator
+          data:
+            deployment-validation-operator-config.yaml: |-
+              checks:
+                # if doNotAutoAddDefaults is true, default checks are not automatically added.
+                doNotAutoAddDefaults: false
+
+                # addAllBuiltIn, if set, adds all built-in checks. This allows users to
+                # explicitly opt-out of checks that are not relevant using Exclude.
+                # Takes precedence over doNotAutoAddDefaults, if both are set.
+                addAllBuiltIn: true
+        - apiVersion: policy.open-cluster-management.io/v1
+          kind: Policy
+          metadata:
+            annotations:
+              policy.open-cluster-management.io/categories: CM Configuration Management
+              policy.open-cluster-management.io/controls: CM-2 Baseline Configuration
+              policy.open-cluster-management.io/standards: NIST SP 800-53
+            name: deployment-validation-operator
+            namespace: openshift-acm-policies
+          spec:
+            disabled: false
+            remediationAction: enforce
+            policy-templates:
+              - objectDefinition:
+                  apiVersion: policy.open-cluster-management.io/v1
+                  kind: ConfigurationPolicy
+                  metadata:
+                    name: deployment-validation-operator
+                  spec:
+                    evaluationInterval:
+                      compliant: 2h
+                      noncompliant: 45s
+                    namespaceSelector:
+                      matchLabels:
+                        hypershift.openshift.io/hosted-control-plane: "true"
+                    object-templates:
+                      - complianceType: MustHave
+                        objectDefinition:
+                          apiVersion: package-operator.run/v1alpha1
+                          kind: Package
+                          metadata:
+                            name: deployment-validation-operator
+                          spec:
+                            image: ${REGISTRY_IMG}@${IMAGE_DIGEST}
```

Comment:
Replace this objects with all the objects you listed before the policy (Namespace, Subscription, Service...)